css/components/modal.css

```diff
@@ -37,8 +37,8 @@
 
```

Comment:
commit message does not specify what kind of changes.

Reply:
i created entire thing from start. I could not come up with a better commit message within character limit.

Reply:
understood.